cuda/test/matrix/dense_kernels.cpp

```diff
@@ -350,10 +387,22 @@ TEST_F(Dense, ConvertToSellpIsEquivalentToRef)
 }
 
 
-TEST_F(Dense, CountNNZIsEquivalentToRef)
+TEST_F(Dense, MoveToSellpIsEquivalentToRef)
 {
     set_up_apply_data();
+    auto sellp_mtx = gko::matrix::Sellp<>::create(ref);
+    auto dsellp_mtx = gko::matrix::Sellp<>::create(cuda);
+
+    x->move_to(sellp_mtx.get());
+    dx->move_to(dsellp_mtx.get());
 
+    GKO_ASSERT_MTX_NEAR(sellp_mtx, dsellp_mtx, 1e-14);
+}
+
+
+TEST_F(Dense, CountNNZIsEquivalentToRef)
+{
+    set_up_apply_data();
     gko::size_type nnz;
     gko::size_type dnnz;
 
```

Comment:
It's for what is down here, but most other tests (at least all CalculateNNZ) do not follow the AAA pattern either.

Reply:
Yes, some other tests do not follow the AAA pattern, but I thought when I edit these files, I can make at least them follow this pattern.
I could also go through all the tests and make them follow this pattern, should I do that?

Reply:
I don't think going through all of Ginkgo is needed right now, but maybe in this file at least since you're modifying it with this PR anyway.

Reply:
Makes sense, will do that.